packages/forms-web-app/package.json

```diff
@@ -52,7 +52,7 @@
 		"nunjucks": "^3.2.3",
 		"nunjucks-date-filter": "^0.1.1",
 		"pino": "^6.13.2",
-		"redis": "3.1.2",
+		"redis": "^4.5.1",
```

Comment:
Do we want to allow it to update compatible version with the ^ ?

Appreciate we are not consistent across the repo and we don't have an npmrc with 'save-exact=true' inside.

Reply:
Happy with that. Will resolve.